src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -567,13 +568,13 @@ private Future<ReceiptStatus> calculatePoLineReceiptStatus(List<Piece> byPoLine,
       return Future.succeededFuture(FULLY_RECEIVED);
     }
     // Partially Received: In case there is at least one successfully received piece
-    if (StreamEx.of(pieces).anyMatch(piece -> ReceivingStatus.RECEIVED == piece.getReceivingStatus())) {
```

Comment:
Did you investigate do we need some changes also in  CheckinHelper, ReceivingHelper, CheckingReceivePoecesHelper, ReOpenCompositeOrderManager
![image](https://github.com/folio-org/mod-orders/assets/25097693/ba1c99e0-b82a-46dd-a59b-cf796e301b73)

Reply:
updated ReOpenCompositeOrderManager